node/src/components/deploy_buffer.rs

```diff
@@ -187,13 +223,14 @@ impl DeployBuffer {
     ) -> HashSet<DeployHash> {
         let past_deploys = past_blocks
             .iter()
-            .filter_map(|block_hash| self.processed.get(block_hash))
+            .filter_map(|block_hash| self.proposed.get(block_hash))
             .chain(self.finalized.values())
             .flat_map(|deploys| deploys.keys())
             .collect::<HashSet<_>>();
-        // deploys_to_return = all deploys in collected_deploys that aren't in finalized blocks or
-        // processed blocks from the set `past_blocks`
-        self.collected_deploys
+
+        // deploys_to_return = all deploys in pending that aren't in finalized blocks or
+        // proposed blocks from the set `past_blocks`
```

Comment:
I think the identifiers in that comment are outdated. Should `past_blocks` be `past_deploys`?

Reply:
`past_blocks` is still the name of the argument used to construct `past_deploys` - a `HashSet<ProtoBlockHash>` while `past_deploys` is a `HashSet<DeployHash>` collected from it.